src/api/app/views/webui/users/tokens/users/index.html.haml

```diff
@@ -1,14 +1,26 @@
 :ruby
   @pagetitle = if @token.description.blank?
-                 'Users of the token'
+                 'Share token'
                else
-                 "Users of the '#{@token.description}' token"
+                 "Share '#{@token.description}' token"
                end
 
 .card.mb-3
   .card-body
+    %h3 Share Your Token with Others
```

Comment:
I think this title is too long, what about `Token Sharing`?

Reply:
Fine. Applied.